src/drivers/uavcan/CMakeLists.txt

```diff
@@ -45,7 +45,9 @@ if(CONFIG_ARCH_CHIP)
 		set(UAVCAN_TIMER 1)
 	elseif(${CONFIG_ARCH_CHIP} MATCHES "stm32h7")
 		set(UAVCAN_DRIVER "stm32h7")
-		set(UAVCAN_TIMER 5) # The default timer is TIM5
+		if (DEFINED config_uavcan_timer_override)
```

Comment:
```suggestion
                              set(UAVCAN_TIMER 5) # The default timer is TIM5		
                              if (DEFINED config_uavcan_timer_override)
```

Reply:
To check your changes. Please build all the H7 targets. 
use `find boards/ -name defconfig | xargs grep CONFIG_ARCH_CHIP_STM32H7=y` to gee the list of what you have to build test.

Reply:
I had changed it agian base on your suggestions ,could you please review it again.

Reply:
Please see why the build fails. Run 'make cubepilot_cubeorange_default' and fix the issues.